.github/workflows/main.yml

```
@@ -25,40 +25,56 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    # needed java 1.11 for checkstyle
```

```suggestion
    # Java 11 is specifically needed for Checkstyle.
```

Reworded, and changed casing for consistency.

---

done

---

resolving on behalf of ben as this is a straight change.